client/utils/account-fees.tsx

```diff
@@ -74,24 +74,27 @@ const getStripeFeeSectionUrl = ( country: string ): string => {
 	);
 };
 
-const getFeeDescriptionString = ( fee: BaseFee ): string => {
+const getFeeDescriptionString = (
+	fee: BaseFee,
+	discountRate: number
+): string => {
 	if ( fee.fixed_rate && fee.percentage_rate ) {
 		return sprintf(
 			'%1$f%% + %2$s',
-			formatFee( fee.percentage_rate ),
-			formatCurrency( fee.fixed_rate, fee.currency )
+			formatFee( fee.percentage_rate * discountRate ),
+			formatCurrency( fee.fixed_rate * discountRate, fee.currency )
 		);
 	} else if ( fee.fixed_rate ) {
 		return sprintf(
 			'%2$s',
-			formatFee( fee.percentage_rate ),
-			formatCurrency( fee.fixed_rate, fee.currency )
+			formatFee( fee.percentage_rate * discountRate ),
```

Comment:
Do we need `formatFee( fee.percentage_rate * discountRate )` here within the `sprintf` statement, for a condition when it is not available?  Will this be better: 

```suggestion
if ( fee.fixed_rate ) {
		return sprintf(
			'%1$s',
			formatCurrency( fee.fixed_rate * discountRate, fee.currency )
		);
```